src/sqlfluff/dialects/dialect_postgres.py

```diff
@@ -358,13 +371,6 @@
         ),
     ),
     FrameClauseUnitGrammar=OneOf("RANGE", "ROWS", "GROUPS"),
-    # In Postgres, column references may be followed by a time zone cast in all cases.
-    # For more information, see
-    # https://www.postgresql.org/docs/11/functions-datetime.html
```

Comment:
This should have been removed as part of #3115 but wasn't (see comment here: https://github.com/sqlfluff/sqlfluff/pull/3115/files#r852459892)